### PR TITLE
Various fixes to the docs

### PR DESCRIPTION
**Pull request by Mario54**
@alavers @dannytranlx @lemieux @jugarrit @mspensieri 

* Add UI to the Web section
* Make it clearer how to login with a JWT
* Update list of supported languages
* Update whispers templates

### CODE REVIEW

**Comment by lemieux** [comment]
:+1:

File: source/includes/users/_users_on_multiple_devices.md [hunk 1]
```diff
@@ -27,27 +27,46 @@ Smooch.login(yourUserId);
 Smooch.login(yourUserId);
```
Comment (mspensieri) [inline source/includes/users/_users_on_multiple_devices.md]: This method does not exist in Java (https://trello.com/c/Tq5m91nZ/2558-docs-android-login-don-t-include-jwt-parameter-example-just-has-smooch-login-userid). Need to do `Smooch.login(yourUserId, null);`

**Comment by mspensieri** [comment]
:+1:

File: source/includes/users/_users_on_multiple_devices.md (hunk 1 above)
Comment (Mario54) [inline source/includes/users/_users_on_multiple_devices.md]: Oh, that's what was meant by the card xD

Alright, I'll fix

Comment (Mario54) [inline source/includes/users/_users_on_multiple_devices.md]: Fixed

**Comment by dannytranlx** [comment]
:+1:

**Comment by alavers** [comment]
:+1:

**Comment by Mario54** [comment]
😊😊😊😱😱😰😯

**Pull request merged**